parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -300,6 +300,16 @@ write_files:
     #EOF
 {{- end}}
 
+{{- if ShouldConfigureCustomCATrust}}
+{{range $i, $cert := GetCustomCATrustConfigCerts}}
+- path: /usr/local/share/ca-certificates/00000000000000cert{{$i}}.crt
```

Comment:
lol

I guess this is to match the regex in our update logic? can you leave a comment ?

{{/ /}} I think in go tmpl

Reply:
Yeah added a comment in PR in generated file explaining that. This is so that these certs get removed once the pod/ds is running and update flow begins.

Will add comment in code too.

Reply:
ah my bad, read the pr sequentially and started commenting :D